### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by fantasysponge** [yjn870/IDN-pytorch#1]
Did you train this code, when I train, I find the loss value is too high, when eliminate the weight initialization function, it gets normal
